.gitignore

```diff
@@ -1,6 +1 @@
-.DS_Store
-.DS_Store?
-.Spotlight-V100
-.Trashes
-Icon
-Icon*
+config/settings
```

Comment:
I addressed this in the README, but IMHO these types of files should be ignored globally - it's basically garbage that *your* OS creates, the project should not be cluttered with this.